.gitignore

```diff
@@ -15,7 +15,7 @@
 ansible/workdir/*
 ansible/judd.sh
 ansible/collections
-ansible/configs/*/roles
+ansible/configs/*/roles/*
```

Comment:
can you explain why this change is needed?

see https://github.com/redhat-cop/agnosticd/blob/development/docs/How_to_use_galaxy_roles_in_your_config.adoc#development-workflow

Reply:
OK  sorry, just saw it in the description

> So while this PR is relatively large it should not touch on anything outside our config with one exception: /.gitignore. As we now need to include roles in our config dir, we put a "sub" .gitignore file into the ansible-automation-platform dir. For this to work we needed to change ignoring ../roles to ../roles/* so we could "un-ignore" in our config dir.

Let me discuss it internally

Reply:
after discussing with @wkulhanek  and @tonykay, we don't think it's a good idea to allow this practice. Documentation about `ansible/configs/*/roles` is pretty clear, it's reserved for dynamic roles imported by ansible galaxy or collections.

You have the choice to:
* move the roles to ansible/roles 
* externalize the roles in another repository and import them using the requirements.txt file in the config directory

Reply:
It is rolled back (just the ending line is still there)